napalm_ios/ios.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, hostname, username, password, timeout=60, optional_args=None)
         self.password = password
         self.timeout = timeout
 
+        self.transport = optional_args.get('transport', 'ssh')
```

Comment:
Ok, sorry about that. Let's call it transport. I will send a PR to update the `eos` driver.